instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/NettyClientTelemetry.java

```diff
@@ -63,4 +66,12 @@ public ChannelInboundHandlerAdapter createResponseHandler() {
       createCombinedHandler() {
     return new HttpClientTracingHandler(instrumenter);
   }
+
+  /**
+   * Propagate the current context to the {@link Channel}. This MUST be called before each HTTP
+   * request executed on a {@link Channel}.
+   */
+  public static void setChannelContext(Channel channel) {
+    channel.attr(AttributeKeys.WRITE_CONTEXT).compareAndSet(null, Context.current());
+  }
```

Comment:
probably good to pass in `Context` as arg, for people who don't use thread-bound Context (e.g. some reactive apps)

also, then we could use this method from javaagent instrumentation too (e.g. reactor-netty)

Reply:
Ok, will start with one variant that accepts `Context` as an arg. Can add an overload where the context defaults to `Context.current()` later if needed.